app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,42 @@
+class FoodsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @foods = current_user.foods
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    @food = current_user.foods.new(food_params)
+
+    if @food.save
+      redirect_to foods_url, notice: 'Food was successfully created.'
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
```

Comment:
- [ ] Good job so far you app works without errors :clap: But I can not delete a food item from the lists as owner . When I clicked on delete it return a blank page with the item id in the url. Please fix it in order to make your app works as expected :+1:

Reply:
Done

Reply:
Done